config/initializers/deprecations.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
+deprecation = ActiveSupport::Deprecation.new("", "Errbit")
```

Comment:
I was trying to specify Errbit::Version.to_s here but I got an error.

Reply:
1. We don't need version here
2. Rename `deprecation` to `deprecator`
3. First argument should be nil (instead of empty string)

Fix all this and I will merge. And backport to 0-10-stable branch.